pkg/api/server/v1alpha2/results.go

```diff
@@ -67,6 +68,34 @@ func (s *Server) GetResult(ctx context.Context, req *pb.GetResultRequest) (*pb.R
 	return result.ToAPI(store), nil
 }
 
+// UpdateResult receives Result and FieldMask from client and uses them to update records in local Sqlite Server.
```

Comment:
```suggestion
// UpdateResult updates a Result in the database.
```

Figured this is copied from v1alpha1, but we're not updating records here, fieldmask isn't supported yet, and the real deployment doesn't use sqlite. 😅

Reply:
We should also add FieldMask as a TODO (or perhaps since we're not actually using this in a meaningful way, just remove it from the proto altogether).

Reply:
Okay, I'm going to move the fieldmask field